src/lib/src/K8sService.js

```diff
@@ -0,0 +1,72 @@
+const assert = require('assert');
+const K8sApi = require('kubernetes-client');
+const {Client, config} = K8sApi;
+
+const FlowCRD = require('../data/FlowCRD.json');
+const SchedulerCRD= require('../data/SchedulerCRD.json');
+
+class K8sService{
+    constructor(app) {
+        this._app = app;
+    }
+
+    async start() {
```

Comment:
Looks to me more like `init`, cause it doesn't really establish any connection or something like that.